<ui version="4.0" >
 <class>TTProgressForm</class>
 <widget class="QDialog" name="TTProgressForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>480</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Progress Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLabel" name="actionString" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="laElapsedTime" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Time elapsed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="elapsedTimeString" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>13</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" colspan="2" >
       <widget class="QLabel" name="laPercentages" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Complete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <widget class="QLabel" name="percentageString" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>0%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6" >
       <widget class="QProgressBar" name="progressBar" >
        <property name="value" >
         <number>24</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="6" colspan="2" >
       <widget class="QPushButton" name="pbCancel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>471</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="6" colspan="2" >
       <widget class="QCheckBox" name="cbDetails" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Show details</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QScrollArea" name="scrollArea" >
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>578</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="0" >
        <layout class="QVBoxLayout" name="verticalLayout" />
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>392</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
